istioctl/pkg/auth/upgrader.go

```diff
@@ -69,6 +69,27 @@ const (
 	requestAuthPrincipal = "request.auth.principal"
 )
 
+const (
+	rbacGlobalAllowAll = `apiVersion: security.istio.io/v1beta1
```

Comment:
Do you need `---` in the beginning?

Reply:
No, since clusterrbacconfig is converted first, we only need to append each equivalent AuthorizationPolicy with ---.